src/app/Topology/Shared/utils.tsx

```diff
@@ -105,13 +105,26 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
           NodeStatus.warning,
           {
             title: 'Failed to compute JVM ID',
-            description: `Target ${node.target.alias} might be missing credentials.`,
+            description: (
+              <>
+                <Text component={TextVariants.p}>
+                  If target has JMX Authentication enabled, add the credential to Cryostat keyring.
+                </Text>
+                <Text component={TextVariants.p}>
+                  If the target has SSL enabled over JMX, add its certificate to Cryostat truststore. Restart is
+                  required.
+                </Text>
+              </>
+            ),
             callForAction: [
               <WarningResolverAsCredModal key={`${node.target.alias}-resolver-as-credential-modal`}>
                 <Button variant="link" isSmall style={{ padding: 0 }}>
                   Add credentials
                 </Button>
               </WarningResolverAsCredModal>,
+              <WarningResolverAsLink key={`${node.target.alias}-resolver-as-link-to-security`} to="/security">
+                Add Certificates
```

Comment:
`credentials` vs `Certificates` - they should have consistent capitalization. I'm not sure which is preferred for text action buttons like this.